File: Syncfusion_Sample/Windows/Grid.Windows/Samples/2.0/Product Showcase/Pivot Grid Demo/cs/PivotGridControl/IconDrawingSupport.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using System.Drawing.Imaging;

using Syncfusion.Windows.Forms.Grid;
using Syncfusion.Drawing;

namespace PivotGridLibrary
{
    public class IconDrawingSupport
    {

        static GridIconPaint iconPainter;

        static IconDrawingSupport()
		{
            iconPainter = new GridIconPaint("PivotGridControl.", typeof(IconDrawingSupport).Assembly);
		}

        public static void DrawImage(Graphics g, string imageFileName, Rectangle rect, Point offSet)
        {
            string prefix = "icons."; //folder in project where the bitmaps are located
            string bitmapName = prefix + imageFileName;

            iconPainter.PaintIcon(g, rect, offSet, bitmapName, Color.Black);
        }

        /// <summary>
        /// Bitmap for moving rows.
        /// </summary>
        public static Bitmap RedLeftBitmap
        {
            get
            {
                return iconPainter.GetBitmap("icons.RedLeftArrow.bmp");
            }
        }

        /// <summary>
        /// Bitmap for moving rows.
        /// </summary>
        public static Bitmap RedRightBitmap
        {
            get
            {
                return iconPainter.GetBitmap("icons.RedRightArrow.bmp");
            }
        }

    }

}

File: Syncfusion_Sample/Windows/Grid.Windows/Samples/2.0/Product Showcase/Pivot Grid Demo/vb/PivotGridControl/IconDrawingSupport.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Windows.Forms
Imports System.Drawing.Imaging

Imports Syncfusion.Windows.Forms.Grid
Imports Syncfusion.Drawing

Namespace PivotGridLibrary
	Public Class IconDrawingSupport

		Private Shared iconPainter As GridIconPaint

		Shared Sub New()
			iconPainter = New GridIconPaint("PivotGridControl.", GetType(IconDrawingSupport).Assembly)
		End Sub

		Public Shared Sub DrawImage(ByVal g As Graphics, ByVal imageFileName As String, ByVal rect As Rectangle, ByVal offSet As Point)
			Dim prefix As String = "icons." 'folder in project where the bitmaps are located
			Dim bitmapName As String = prefix & imageFileName

			iconPainter.PaintIcon(g, rect, offSet, bitmapName, Color.Black)
		End Sub
	End Class

End Namespace
